docs/template/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
ISSUE_TEMPLATE.md and PULL_REQUEST_TEMPLATE.md must be `/.github`, so that github automatically generates template when user files an issue or submits a pull request.
@garfield009 @maomaode